Foundation/include/Poco/ClassLibrary.h

```diff
@@ -67,10 +67,10 @@ extern "C"	\
 #define POCO_BEGIN_MANIFEST_IMPL(fnName, base) \
 	bool fnName(Poco::ManifestBase* pManifest_)										\
 	{																				\
-		typedef base _Base;															\
-		typedef Poco::Manifest<_Base> _Manifest;									\
-		std::string requiredType(typeid(_Manifest).name());							\
-		std::string actualType(pManifest_->className());							\
+		using _Base = base;															\
```

Comment:
we should change this naming, underscore and capital first letter are [reserved for the implementation](https://devblogs.microsoft.com/oldnewthing/20230109-00/?p=107685)